packages/merge-styles/etc/merge-styles.api.md

```diff
@@ -38,7 +38,7 @@ export function fontFace(font: IFontFace): void;
 
 // @public
 export type IConcatenatedStyleSet<TStyleSet extends IStyleSet<TStyleSet>> = {
-    [P in keyof Omit<TStyleSet, 'subComponentStyles'>]: IStyle;
+    [P in keyof Omit_2<TStyleSet, 'subComponentStyles'>]: IStyle;
```

Comment:
This one is probably because there's an `Omit` built in to TS 3.5+, and `merge-styles` also declares a (slightly different) type called `Omit` from back before the built-in one existed. I'm not sure why the emit here changed between TS and/or API Extractor versions, but it's probably not an issue.